02-about.Rmd

```diff
@@ -0,0 +1,80 @@
+
+AP gives analysts the tools to develop reproducible analytical pipelines (RAPs) to automate time–consuming and repetitive tasks, allowing them to focus on interpreting the results.
+
+Reproducible analytical pipelines are achieved in AP with three elements:
+
+* datasets are versioned - snapshots of the data are taken when imported into AP with automated pipelines
+* code is versioned - in GitHub
+* system libraries are standardized - a standardized virtual machine running R Studio/Jupyter, or code running in an explicitly defined Dockerfile
+
+GitHub provides a single location for all of our code -- this enables analysts to collaborate more effectively, share knowledge, do peer review and produce high–quality reproducible analysis
+
+## Secure and well-engineered platform
+
+AP is built in a cloud–based ecosystem that is easy to access remotely from all MoJ IT systems. It provides analysts with access to powerful computational resources that can be scaled to meet demand -- this allows analysts to quickly and easily work with big data and perform complex analytical tasks
+
+AP is designed for data at security classification OFFICIAL and has successfully gained assurance for significant datasets marked OFFICIAL-SENSITIVE. It follows NCSC Cloud Security Principles, implementing features such as:
+
+* 2 factor authentication for user sign-in
+* encryption of data at rest and in transit
+* fine-grained access control
+* extensive tracking of user behaviour, user privilege requests/changes and data flows
+* multiple levels of isolation between users and system components
+  
+Resilience and high availability means analysts can depend on AP to store their work and data, and to complete time–sensitive and business–critical projects.
+
+## What it isn't for
+
+AP does a lot, but data is a big subject and AP doesn't try to do everything. These things are currently outside scope:
+
+### Production apps at scale
+
+Apps on AP should be restricted to private prototypes or for small numbers of staff users (e.g. less than 50). A key benefit of AP is that analysts and data scientists have all the tools to be creative and experimental with interactive dashboards and apps, in a safe sandbox environment. In addition, simple apps for a few staff e.g. for searching through data are fine. However the AP team aims to rapidly change and improve the apps deployment and hosting, so sets an apps availability expectation of only 95% at this stage. Once an app is proven as an AP prototype then it should be adopted by a MoJ Digital & Technology service team to be developed by a multi-disciplinary team and meet the Service Standard.
```

Comment:
Is it worth adding that we won't do weekend and evening support?
Do we want to also explicitly prohibit line-of-business stuff, and stuff going to the public?
Is 50 the right number?